ui/packages/catalog-ui-search/src/main/webapp/component/list-interactions/list-interactions.view.js

```diff
@@ -56,10 +63,14 @@ module.exports = Marionette.ItemView.extend({
         var newList = new this.model.constructor(copyAttributes);
         this.model.collection.add(newList);
     },
+    triggerAction: function(event){
+       var url = event.currentTarget.getAttribute('data-url');
```

Comment:
New JavaScript code should use ES6 features; `const` should be used here instead of `var`.